exercises/affine-cipher/affine_cipher_test.py

```diff
@@ -0,0 +1,68 @@
+import unittest
+
+from affine_cipher import decode, encode
+
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.0
+
+class AffineCipherTest(unittest.TestCase):
+    def test_encode_yes(self):
+        self.assertMultiLineEqual(encode("yes", 5, 7), "xbt")
```

Comment:
Unless a string is expected to contain multiple lines, it is probably simpler to just use `assertEqual`.